src/Cms/AppErrors.php

```diff
@@ -158,9 +158,27 @@ protected function setWhoopsHandler($handler): void
         $whoops = $this->whoops();
         $whoops->clearHandlers();
         $whoops->pushHandler($handler);
+        $whoops->pushHandler($this->getExceptionHookWhoopsHandler());
         $whoops->register(); // will only do something if not already registered
     }
 
+    /**
+     * Initialize a callback handler for triggering the system.exception hook
+     *
+     * @return CallbackHandler
```

Comment:
Minor consistency fixes:

```suggestion
     * Initializes a callback handler for triggering the `system.exception` hook
     *
     * @return \Whoops\Handler\CallbackHandler
```